change/@ni-nimble-components-c605be0d-27fc-47ff-bf8e-c996c0fe7158.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Fix intermittent drawer test",
```

Comment:
nit: can you update the PR title to be more descriptive? This comment would work